notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -q "optimum[openvino]&gt;=1.8" "diffusers&gt;=0.16.1 transformers&gt;=4.28.0"
</pre><p>!pip install -q "optimum[openvino]&gt;=1.8" "diffusers&gt;=0.16.1" "transformers&gt;=4.28.0"</p><p> </p><p>now for me it cause error</p><pre class="ql-syntax" spellcheck="false">ERROR: Invalid requirement: 'diffusers&gt;=0.16.1 transformers&gt;=4.28.0'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>